src/main-app.js

```diff
@@ -211,9 +211,15 @@ export class MainApplication extends LitElement {
       this._actionBtn.textContent = "";
     });
 
-    this._store.addEventListener('change', ev => {
-      this._hideScanInfo = true;
-    }, { once: true });
+    const onchange = async () => {
+      for await (let entry of this._store.entries()) {
+        this._hideScanInfo = true;
+        this._store.removeEventListener('change', onchange);
```

Comment:
why is this removed multiple times?

Reply:
Why dont you not just modify this

![image](https://user-images.githubusercontent.com/1101339/70313278-9451f380-1815-11ea-84a3-38dc13b5d7ed.png)

just add the this._hideScanInfo = true as the first line of the for await loop

Reply:
like inside the if(first)

Reply:
Looks like the scope of `_hideScanInfo` property is limited in main.js, that's why I have to add an `onchange` here.

Reply:
> why is this removed multiple times?

Two scenarios here,
1. Reload page with existing cached store items, we invoke `onchange` directly to hide the scan info.
2. Load page with no cached store items, when new item is added, `change` event listener triggers to hide the scan info.